O/OpenBLAS/OpenBLAS@0.3.21/build_tarballs.jl

```diff
@@ -9,7 +9,7 @@ version = v"0.3.21"
 sources = openblas_sources(version)
 script = openblas_script(;aarch64_ilp64=true, num_64bit_threads=512)
 platforms = openblas_platforms(;experimental=true)
-push!(platforms, Platform("x86_64", "linux"; sanitize="memory"))
+push!(platforms, Platform("x86_64", "linux"; sanitize="memory", libgfortran_version=v"5.0.0"))
```

Comment:
But this wouldn't work for other libgfortran versions, which is false. Maybe this should be fixed in the Julia side?

Reply:
I'm not sure if the other libfortran versions would be useful when building with msan since for now we are only supporting one platform, but I guess it doesn't cost much to add them all in any case.

Reply:
I'm ***not*** suggesting to add two more platforms which would result in identical builds, which is quite a waste of resources

Reply:
The issue is that the bb install script is kinda hardcoded, so I guess make it so, if there is a sanitize flag on, do not check for libgfortran/cxxabi stuff?

Reply:
> The issue is that the bb install script is kinda hardcoded

That's exactly what I suggested to fix by saying

> Maybe this should be fixed in the Julia side?

Reply:
This wasn't as bad as I expected, so it's unnecessary to build another version.